bib/cmd/bootc-image-builder/main_test.go

```diff
@@ -62,16 +64,20 @@ type manifestTestCase struct {
 }
 
 func getBaseConfig() *main.ManifestConfig {
-	return &main.ManifestConfig{Imgref: "testempty"}
+	return &main.ManifestConfig{
+		Architecture: arch.Current(),
```

Comment:
Hmm, this makes the tests behave differently on each arch... Not sure if that's conceptually OK, or not. :thinking:

Reply:
I have no strong opinion here, I moved to a static arch now.